TestPlaybooks/playbook-DUO-Test.yml

```
@@ -0,0 +1,342 @@
+id: duo_test
```

Just a convention we are trying to make, btw it should be automatically generated to be identical as the name, so you are probably not pulled in the server/web-client:
```suggestion
id: DUO Test Playbook
```